openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
At the moment, the user gets also all permissions from the Registered group. But in a few seconds/hours we will merge the Pull request of @FinnStutzenstein with the permission matrix and some changes on the built-in groups. Then your workflow should work as expected. So just wait on #2260 .